packages/mobile/src/screens/edit-track-screen/screens/RemixSettingsScreen.tsx

```diff
@@ -206,6 +211,13 @@ export const RemixSettingsScreen = () => {
       title={messages.screenTitle}
       icon={IconRemix}
       variant='white'
+      topbarLeft={
+        <TopBarIconButton
+          icon={IconCaretLeft}
+          style={styles.backButton}
+          onPress={hasErrors ? undefined : handleSubmit}
```

Comment:
it's a little hacky but good enough for now. i think the best would be a button with a disabled prop instead of overriding the on submit

Reply:
yeah. we've used this pattern in a few other places. we should be updating all of those together when we retouch them